local-cli/rnpm/core/__tests__/getCommands.spec.js

```diff
@@ -0,0 +1,172 @@
+// @todo replace rewire and mock-require
+// jest.autoMockOff();
+//
+// const path = require('path');
+// const mock = require('mock-require');
+// const rewire = require('rewire');
```

Comment:
@Kureev we need to replace rewire and mock-require right after that, are you up to giving it a go?